_test/test_change_crystal/test_change_crystal_bragg_coarse.m

```diff
@@ -273,6 +273,46 @@ function test_finalize_alignment_on_file_keep(obj)
         end
         %------------------------------------------------------------------
         %------------------------------------------------------------------
+        function test_revert_alignment_works_on_file_and_on_filebacked(obj)
+            corr  = crystal_alignment_info([5.0191 4.9903 5.0121], ...
+                [90.1793 90.9652 89.9250], [-0.0530 0.0519 0.0345]);
+            test_file_fb = build_tmp_file_name(obj.misaligned_sqw_file);
+            copyfile(obj.misaligned_sqw_file, test_file_fb, 'f');
+            test_file_file = fullfile(tmp_dir,'misaligned_file_to_check_reversion.sqw');
+            copyfile(obj.misaligned_sqw_file, test_file_file, 'f');
+            clOb = onCleanup(@()delete(test_file_file));
+
+
+            test_fb = sqw(test_file_fb,'file_backed',true);
+            assertTrue(test_fb.is_filebacked, ...
+                'This test validates filebacked operations but the object test_fb placed in memory')
+            test_fb = test_fb.set_as_tmp_obj();
+            assertTrue(test_fb.is_tmp_obj)
+
+
+            % TEST:
+            % apply alignment on FB object
+            test_fb_ref = change_crystal(test_fb, corr);
```

Comment:
be nice if line 277 were placed just before this one so we can see quickly what corr means